src/navigator-card-body.jsx

```diff
@@ -88,11 +88,7 @@ const ContextMenuItems = ({ path, currentDir, selected, setSelected, setHistory,
 
     const _createDirectory = () => createDirectory(Dialogs, currentDir);
     const _createLink = () => createLink(Dialogs, currentDir, files, selected[0]);
-    const _copyItem = () => {
-        copyItem(setClipboard, selected.length > 1
-            ? selected.map(s => currentDir + s.name)
-            : [currentDir + selected[0].name]);
-    };
+    const _copyItems = () => copyItems(setClipboard, selected.map(s => currentDir + s.name));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-254-20240212-112317-82181124-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)